### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by maleck13**
update verification to reflect the new process

## Is an upgrade task required and are there additional steps needed to test this?

- [ ] Yes
- [ ] No

- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

### CODE REVIEW

**Comment by maleck13** [comment]
@david-martin  @davidffrench  when you get a min

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 <!-- Add any additional information needed. Such as the Jira or GH issue this PR relates to or any other context you feel is necessary.) -->
 
 ## Verification Steps
+As the verifier of the PR the following process should be done:
+
+- Ensure the developer of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
+- Ask the developer do they have a cluster on which you can verify the fresh installation is correct.
+- Ask the developer to run the upgrade and attach an upgrade log. Ensure and that it exited as expected. If possible look at the tasks that ran and see they match the PR
```
Comment (david-martin) [inline .github/PULL_REQUEST_TEMPLATE.md]: @maleck13 If i'm understanding this right, we're saying he verification no longer requires the verifier to try out the code changes. It is enough for the PR creater to produce proof that it was run?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 <!-- Add any additional information needed. Such as the Jira or GH issue this PR relates to or any other context you feel is necessary.) -->
 
 ## Verification Steps
+As the verifier of the PR the following process should be done:
+
+- Ensure the developer of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
+- Ask the developer do they have a cluster on which you can verify the fresh installation is correct.
```
Comment (davidffrench) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- Verify the fresh installation is correct on cluster provided by PR author 
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 <!-- Add any additional information needed. Such as the Jira or GH issue this PR relates to or any other context you feel is necessary.) -->
 
 ## Verification Steps
+As the verifier of the PR the following process should be done:
+
+- Ensure the developer of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
```
Comment (davidffrench) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
```

Comment (davidffrench) [reply .github/PULL_REQUEST_TEMPLATE.md]: Can we provide steps somewhere else on how to write the installation run output to a file

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (davidffrench) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 
```

Comment (maleck13) [reply .github/PULL_REQUEST_TEMPLATE.md]: yes then the verifier should inspect the cluster to ensure it ended in the correct state

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (maleck13) [reply .github/PULL_REQUEST_TEMPLATE.md]: well it is just logged to std out so ```> tofile.txt ``` is all that is needed

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -2,6 +2,18 @@
...
+
+### Installation Verification
+- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
+- Verify the fresh installation is correct on cluster provided by PR author 
+### Upgrade Verification
+- After installation verification, notify the PR author to begin an upgrade on their cluster
+- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
+- If possible, look at the tasks that ran and see they match the PR
+- Verify the upgrade is correct on cluster provided by PR author 
+- Re check the cluster and ensure the changes expected happend after the upgrade.
```
Comment (davidffrench) [inline .github/PULL_REQUEST_TEMPLATE.md]: Only need one of these lines.

Other than that, the PR is good to go.

**Pull request merged**